xonsh/environ.py

```diff
@@ -1708,6 +1699,18 @@ def DEFAULT_VARS():
             "Whether or not to store the ``stdout`` and ``stderr`` streams in the "
             "history files.",
         ),
+        "XONSH_STYLE_OVERRIDES": Var(
```

Comment:
Should this apply to readline & ansi_colors too then?

Reply:
That was the plan, I just forgot it, will try to do it.

Reply:
Now it does.